gfx.cpp

```diff
@@ -471,6 +472,16 @@ class GfxInternal
             GfxRaytracingPrimitive parent_ = {};
         }
         instance_;
+        struct
+        {
+            uint32_t build_flags_ = 0;
+            GfxBuffer bvh_buffer_ = {};
+            uint64_t bvh_data_size_ = 0;
+            uint32_t aabb_stride_ = 0;
+            GfxBuffer aabb_buffer_ = {};
+            GfxAccelerationStructure acceleration_structure_ = {};
+        }
+        aabb_;
```

Comment:
I think the name of this should be `procedural_` instead of `aabb_`. The other structs above are all named after the corresponding type in the enum {Triangles, Instance, Procedural}. Also its more than just a standard AABB so best to avoid confusion with the name.

Reply:
Done